client/src/components/Ticketing/cart/Cart.tsx

```diff
@@ -112,21 +155,41 @@ const Cart = () => {
               flex-col items-center rounded-xl justify-between'>
               <div className='flex flex-col items-center'>
                 <div className='text-zinc-100 text-xl font-semibold'>Subtotal</div>
-                <div className='text-white'>{toDollarAmount(subtotal)}</div>
+                <div className='text-amber-300 italic'>{printDiscountText(discount)}</div>
+                <div className='text-white'>{toDollarAmount(total)}</div>
               </div>
 
 
               <div className='flex flex-col items-center gap-3 '>
 
                 <div className='flex flex-col items-center form-control disabled:opacity-50 '>
                   <div className='input-group flex flex-row items-center w-full px-3 py-1 text-black rounded-xl bg-sky-500'>
-                    <input type="text" placeholder="Discount code..." className='input input-bordered rounded-md pl-2' />
-                    <button className='btn btn-square bg-sky-500 ml-1'>
-                      <svg xmlns="http://www.w3.org/2000/svg" className="h-6 w-6" fill="none" viewBox="0 0 24 24"
-                        stroke="white"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2"
-                          d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" /></svg>
-                    </button>
+                    <input type="text" placeholder="Discount code..."
+                      className='input input-bordered rounded-md pl-2'
+                      value={(discountText) ? discountText : discount.code}
+                      onChange={(e) => {
+                        setDiscountText(e.target.value.toUpperCase());
```

Comment:
The user input probably doesn't need to be capitalized by default and discount codes can be case insensitive. Just an aesthetic suggestion though so not a blocker for merging the pr and is a relatively small change if PPH want something different